### DISCUSSION TITLE
Load best model checkpoint

### DISCUSSION THREAD

**Discussion by Pacman1984** [PyTorchLightning/lightning-flash#738] [Q&A] [answered]
Hi everyone, 

is flash loading the best model checkpoint after training for max_epochs (best practice) or does it train max_epochs and holds the last model checkpoint?

Thanks and all the best
Sebastian

**Comment by aniketmaurya** [comment, marked as answer]
You can use model checkpoint callback and load the best weight.

```python
from pytorch_lightning.callbacks import ModelCheckpoint

checkpoint_callback = ModelCheckpoint(...)
trainer = Trainer(callbacks=[checkpoint_callback])
trainer.fit(model)

checkpoint_callback.best_model_path

```
